src/wrapper/jnienv.rs

```diff
@@ -1994,7 +1994,7 @@ impl<'a> JNIEnv<'a> {
     ) -> Result<AutoByteArray> {
         non_null!(array, "get_byte_array_elements array argument");
         let mut is_copy: jboolean = 0xff;
-        let ptr = jni_non_void_call!(self.internal, GetByteArrayElements, array, &mut is_copy);
+        let ptr = jni_unchecked!(self.internal, GetByteArrayElements, array, &mut is_copy);
```

Comment:
If we replace jni_non_void_call that checks exceptions with jni_unchecked, we have to check the result to be non-null:

> RETURNS:
> Returns a pointer to the array elements, or NULL if the operation fails.
> THROWS:
> OutOfMemoryError: if the system runs out of memory.

Reply:
We're checking, because we're using `NonNull` wrapper inside `AutoArray`, so this is fine, and it will be more efficient.

Reply:
Or do you prefer to keep `GetXArrayElements` with `jni_non_void_call!`?

I've seen that you made a request for changes, but I'm not sure which ones.

Reply:
Sorry, `jni_unchecked` is fine if the result is checked, I missed that code. Thanks for clarifications!